src/main/java/com/discordbolt/api/command/CommandListener.java

```diff
@@ -25,18 +25,19 @@ public void onMesageEvent(MessageReceivedEvent e) {
             return;
         }
 
-        // Message is just a single prefix.
-        if (message.length() <= 1) {
+        // Message is shorter than or equal to prefix.
+        if (message.length() <= manager.getCommandPrefix(e.getGuild()).length()) {
             return;
         }
 
-        // Check if message started with our command prefix
-        if (message.charAt(0) != manager.getCommandPrefix(e.getGuild())) {
+        // Check if message starts with our command prefix
+        if(!message.startsWith(manager.getCommandPrefix(e.getGuild()))) {
```

Comment:
All prefixes are converted to lowercase so we need to convert the message to lowercase as well.